docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -163,13 +145,13 @@ after adding blobs
 
 ## Flashing
 
-Following procedure will flash Dell OptiPlex 7010/9010 SFF Dasharo firmware to your
-machine SPI NOR flash.
+Following procedure will flash Dell OptiPlex 7010/9010 SFF Dasharo firmware to
+your machine SPI NOR flash.
 
-<!-- Link should be replaced to something that not point to 3mdeb cloud. Most
-useful would be location at http server -->
+<!-- Link should be replaced with something that does not point to the 3mdeb cloud. The most
+useful location would be at some HTTP server -->
 
-Flash it on you Dell OptiPlex machine:
+Flash it on your Dell OptiPlex machine:
 
 ``` console
 sudo flashrom -p internal --ifd -i bios -i me -w <dasharo_optiplex_9010_firmware>
```

Comment:
@pietrushnic You claim to use DTS, so all `sudo` stuff should be gone?

Reply:
@macpijan fixed https://github.com/Dasharo/docs/pull/139/commits/d106177e0646625cada9567358b37133fb0f5174